core/payment/src/cli.rs

```diff
@@ -246,17 +249,56 @@ impl PaymentCli {
                     .await?,
                 )
             } // TODO: Uncomment when operation is supported by drivers
-              // PaymentCli::Transfer {
-              //     account,
-              //     driver,
-              //     network,
-              //     to_address,
-              //     amount
-              // } => {
-              //     let address = resolve_address(account).await?;
-              //     let amount = BigDecimal::from_str(&amount)?;
-              //     CommandOutput::object(wallet::transfer(address, to_address, amount, driver, network, token).await?)
-              // }
+            // PaymentCli::Transfer {
+            //     account,
+            //     driver,
+            //     network,
+            //     to_address,
+            //     amount
+            // } => {
+            //     let address = resolve_address(account).await?;
+            //     let amount = BigDecimal::from_str(&amount)?;
+            //     CommandOutput::object(wallet::transfer(address, to_address, amount, driver, network, token).await?)
+            // }
+            PaymentCli::Drivers => {
+                let drivers = bus::service(pay::BUS_ID).call(pay::GetDrivers {}).await??;
+                if ctx.json_output {
+                    return CommandOutput::object(drivers);
+                }
+                Ok(ResponseTable { columns: vec![
+                        "driver".to_owned(),
+                        "recv init?".to_owned(),
```

Comment:
`recv init` is a highly technical details, most probably not interesting to the user.